### ISSUE TITLE
Langchain Quickstart notebook does not work

### ISSUE THREAD

**Issue by mkhammoud** [truera/trulens#1179]
**Bug Description**
Whenever I try to run the [langchain quick start,](https://www.trulens.org/trulens_eval/getting_started/quickstarts/langchain_quickstart/) I get the following error: trulens_eval.feedback.feedback.InvalidSelector: Selector __record__.app.first.steps__.context.first.get_relevant_documents.rets does not exist in source data. 

The RAG itself is perfectly working fine and I get the UI and I can see the evaluation of the relevance metric only. While nothing for context relevance or groundedness. Also in the UI I can see the retrieved context

**To Reproduce**
Running the code without any single change and I still get this error. 

**Expected behavior**
Expecting to have groundedness and qs / context relevance results.

**Relevant Logs/Tracebacks**
Exception in thread Thread-6 (_future_target_wrapper):
Traceback (most recent call last):
  File "C:\Users\mohkh\AppData\Local\Programs\Python\Python311\Lib\site-packages\trulens_eval\feedback\feedback.py", line 1071, in _extract_selection
    arg_vals[k] = list(q.get(source_data))
                  ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\mohkh\AppData\Local\Programs\Python\Python311\Lib\site-packages\trulens_eval\utils\serial.py", line 1034, in get
    for start_selection in start_items:
  File "C:\Users\mohkh\AppData\Local\Programs\Python\Python311\Lib\site-packages\trulens_eval\utils\serial.py", line 1035, in get
    for last_selection in last_step.get(start_selection):
  File "C:\Users\mohkh\AppData\Local\Programs\Python\Python311\Lib\site-packages\trulens_eval\utils\serial.py", line 446, in get
    raise KeyError(
KeyError: 'Key not in dictionary: get_relevant_documents'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\mohkh\AppData\Local\Programs\Python\Python311\Lib\threading.py", line 1045, in _bootstrap_inner
    self.run()
  File "C:\Users\mohkh\AppData\Local\Programs\Python\Python311\Lib\site-packages\ipykernel\ipkernel.py", line 761, in run_closure
    _threading_Thread_run(self)
  File "C:\Users\mohkh\AppData\Local\Programs\Python\Python311\Lib\threading.py", line 982, in run
    self._target(*self._args, **self._kwargs)
  File "C:\Users\mohkh\AppData\Local\Programs\Python\Python311\Lib\site-packages\trulens_eval\utils\python.py", line 475, in _future_target_wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\mohkh\AppData\Local\Programs\Python\Python311\Lib\site-packages\trulens_eval\app.py", line 614, in _manage_pending_feedback_results
    record.wait_for_feedback_results()
  File "C:\Users\mohkh\AppData\Local\Programs\Python\Python311\Lib\site-packages\trulens_eval\schema\record.py", line 186, in wait_for_feedback_results
    feedback_result = future_result.result()
                      ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\mohkh\AppData\Local\Programs\Python\Python311\Lib\concurrent\futures\_base.py", line 449, in result
    return self.__get_result()
           ^^^^^^^^^^^^^^^^^^^
  File "C:\Users\mohkh\AppData\Local\Programs\Python\Python311\Lib\concurrent\futures\_base.py", line 401, in __get_result
    raise self._exception
  File "C:\Users\mohkh\AppData\Local\Programs\Python\Python311\Lib\concurrent\futures\thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\mohkh\AppData\Local\Programs\Python\Python311\Lib\site-packages\trulens_eval\utils\python.py", line 475, in _future_target_wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\mohkh\AppData\Local\Programs\Python\Python311\Lib\site-packages\trulens_eval\utils\threading.py", line 174, in _run_with_timeout
    raise e
  File "C:\Users\mohkh\AppData\Local\Programs\Python\Python311\Lib\site-packages\trulens_eval\utils\threading.py", line 159, in _run_with_timeout
    res: T = fut.result(timeout=timeout)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\mohkh\AppData\Local\Programs\Python\Python311\Lib\concurrent\futures\_base.py", line 456, in result
    return self.__get_result()
           ^^^^^^^^^^^^^^^^^^^
  File "C:\Users\mohkh\AppData\Local\Programs\Python\Python311\Lib\concurrent\futures\_base.py", line 401, in __get_result
    raise self._exception
  File "C:\Users\mohkh\AppData\Local\Programs\Python\Python311\Lib\concurrent\futures\thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\mohkh\AppData\Local\Programs\Python\Python311\Lib\site-packages\trulens_eval\tru.py", line 435, in run_and_call_callback
    temp = ffunc.run(app=app, record=record)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\mohkh\AppData\Local\Programs\Python\Python311\Lib\site-packages\trulens_eval\feedback\feedback.py", line 839, in run
    raise e
  File "C:\Users\mohkh\AppData\Local\Programs\Python\Python311\Lib\site-packages\trulens_eval\feedback\feedback.py", line 825, in run
    input_combinations = list(
                         ^^^^^
  File "C:\Users\mohkh\AppData\Local\Programs\Python\Python311\Lib\site-packages\trulens_eval\feedback\feedback.py", line 1073, in _extract_selection
    raise InvalidSelector(
trulens_eval.feedback.feedback.InvalidSelector: Selector __record__.app.first.steps__.context.first.get_relevant_documents.rets does not exist in source data.

**Environment:**
 - OS: windows
 - Python Version : 3.11.6
 - TruLens version: 0.30.1
 - Versions of Langchain: 0.2.2

**Comment by sfc-gh-jreini** [comment]
Solved by #1187 (merged to main). You can install it from github:

```
pip uninstall trulens_eval -y # to remove existing PyPI version
pip install git+https://github.com/truera/trulens#subdirectory=trulens_eval
```

**Comment by sfc-gh-jreini** [comment]
Let me know if you have any issues with this!
